grails-app/views/email/_shipmentItemReceived.gsp

```diff
@@ -80,7 +80,7 @@
                             ${receiptItem?.inventoryItem?.product?.productCode}
                         </td>
                         <td>
-                            ${format.product(product: receiptItem?.inventoryItem?.product) }
+                            <format:product product="${receiptItem?.inventoryItem?.product}" />
```

Comment:
The `${format.product ...}` call will escape the contents of the GString due to the default codec. But! this call invokes a taglib, and, taglibs need to call `encodeAsHTML()` for correctness. You'll double-escape the product name here, leading to confusing presentation.

Fortunately the fix also increases readability: `<format:product ...>` will get escaped exactly once